Examples/Io/Csv/src/CsvBFieldWriter.cpp

```diff
@@ -49,7 +48,7 @@ void CsvBFieldWriter::run(const Config<Coord, Grid>& config,
 
   // Initialize a CSV writer to the specified filename using the specified
   // column names.
-  dfe::io_dsv_impl::DsvWriter<','> writer(fields, config.fileName);
+  Acts::CsvWriter writer(fields, config.fileName);
```

Comment:
are header/source files missing? should this live in `Acts::`?

Reply:
Right I'll move this to the examples namespace.